src/posix/net.cpp

```diff
@@ -524,6 +524,7 @@ void CurlHttpIO::addaresevents(Waiter *waiter)
 #if defined(_WIN32)
     for (auto& mapPair : prevAressockets)
     {
+        // todo: has the socket always been closed at this point anyway? (and, any chance the socket number might already be being reused?
```

Comment:
Let's check it before merging the hotfix, please. It looks like something risky.

Reply:
Ok, this one is checked and adjusted - since c-ares doesn't inform us about socket creations or closures, but only about the set currently in use, we'll leave sockets other that those untouched.   If any of those sockets do set the event, it's only one extra loop.    While looking into this, I've improved the logging of cURL considerably as you'll see in the subsequent changes - and come across a separate issue that I'll make a ticket for.   thanks